docs/view.md

```diff
@@ -81,6 +81,14 @@ For `View` responder props (e.g., `onResponderMove`), the synthetic touch event
 - [`accessibilityElementsHidden`](view.md#accessibilityElementsHidden)
 - [`accessibilityIgnoresInvertColors`](view.md#accessibilityIgnoresInvertColors)
 - [`shouldRasterizeIOS`](view.md#shouldrasterizeios)
+- [`nextFocusDown`](view.md#nextfocusdown)
+- [`nextFocusForward`](view.md#nextfocusForward)
+- [`nextFocusLeft`](view.md#nextfocusleft)
+- [`nextFocusRight`](view.md#nextfocusright)
+- [`nextFocusUp`](view.md#nextfocusleft)
+- [`clickable`](view.md#clickable)
```

Comment:
can you sort this list alphabetically?

Reply:
Just to clarify, do you want me to make everything alphabetical? From lines 50 - 89 there seems to be no particular order.

Reply:
Yes, let's get everything alphabetized.